packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/data/SummaryService.java

```diff
@@ -0,0 +1,34 @@
+package software.uncharted.terarium.hmiserver.service.data;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.micrometer.observation.annotation.Observed;
+import java.util.List;
+import java.util.UUID;
+import org.springframework.stereotype.Service;
+import software.uncharted.terarium.hmiserver.configuration.Config;
+import software.uncharted.terarium.hmiserver.models.dataservice.Summary;
+import software.uncharted.terarium.hmiserver.repository.data.SummaryRepository;
+import software.uncharted.terarium.hmiserver.service.s3.S3ClientService;
+import software.uncharted.terarium.hmiserver.utils.rebac.Schema;
+
+@Service
+public class SummaryService extends TerariumAssetServiceWithoutSearch<Summary, SummaryRepository> {
+	public SummaryService(
+			final ObjectMapper objectMapper,
+			final Config config,
+			final ProjectAssetService projectAssetService,
+			final SummaryRepository repository,
+			final S3ClientService s3ClientService) {
+		super(objectMapper, config, projectAssetService, repository, s3ClientService, Summary.class);
+	}
+
+	public List<Summary> getSummaries(final List<UUID> ids, final Schema.Permission hasReadPermission) {
+		return repository.findAllByIdInAndDeletedOnIsNull(ids);
```

Comment:
There is no reason to delete those, they will be replaced automatically, but never deleted.
I don't know if we need to filter by `AndDeletedOnIsNull`.

Reply:
True - though the rest of our endpoints all check for this as well. Even if unused, I"m thinking to keep it with the delete check just in case?